src/components/user/resources/Hackpack.tsx

```diff
@@ -14,32 +14,35 @@ const Hackpack = ({ text, techs, link, description }: props) => {
     <div className="rounded-xl bg-white p-3">
```

Comment:
remove the outer div, since there's only one child in the div, u can combine the properties of the div and the link tag